acceptance_tests/features/view_reporting_unit_details.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Display reporting unit page
+  As an internal user
+  I need to to be able to view reporting unit details
+  So that I can undertake a number of potential actions related to that reporting unit
+
+  Background: Internal user is already signed in
+    Given the internal user is already signed in
+
+  @us044-RUDetailsPresentedToUser_s01
```

Comment:
Have we got an agreed format for these tags? In it's current form it's really ugly. Just the user story or a simple feature description would be more readable

Reply:
I copied the style that was in the `add_a_survey.feature` file. We should agree on a better format though i agree. Is that work for this PR?